Gemfile

```
@@ -0,0 +1,30 @@
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you want to use a different version, change it below, save the
+# file and run `bundle install`. Run Jekyll with `bundle exec`, like so:
+#
+#     bundle exec jekyll serve
+#
+# This will help ensure the proper Jekyll version is running.
+# Happy Jekylling!
+# gem "jekyll", "~> 4.1.1"
+# This is the default theme for new Jekyll sites. You may change this to anything you like.
+gem "minima", "~> 2.5"
+# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
+# uncomment the line below. To upgrade, run `bundle update github-pages`.
+gem "github-pages", "~> 4.1.1", group: :jekyll_plugins
```

It looks like this bit here is throwing an error when building because github-pages 4.1.1 technically doesnt exist. 
See [github-pages dependency versions](https://pages.github.com/versions/) & [github pages tutorial Step 9](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site-with-jekyll) for more info :).

```suggestion
gem "github-pages", "~> 209", group: :jekyll_plugins
```

Also I know this wasn't technically part of the AC's but don't want you to be hating life trying to figure this bit out later haha.